internal/controllers/operator_controller.go

```diff
@@ -357,6 +367,19 @@ func isBundleDepStale(bd *rukpakv1alpha1.BundleDeployment) bool {
 	return bd != nil && bd.Status.ObservedGeneration != bd.GetGeneration()
 }
 
+// mapBundleMediaTypeToBundleProvisioner maps an olm.bundle.mediatype property to a
+// rukpak bundle provisioner class name that is capable of unpacking the bundle type
+func mapBundleMediaTypeToBundleProvisioner(mediaType string) (string, error) {
```

Comment:
Maybe have this return the bundle and BD provisioner? That way, if we ever have something that needs a non-plain BD provisioner, it slots in really easily.

Reply:
Sounds good! Definitely non-blocking for this PR.